agent/bench-scripts/tests/pbench-user-benchmark/test-03.post

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cp -H ${_tdir}/test-bin/pbench-*-tools ${_testopt}/unittest-scripts/
```

Comment:
Isn't this fragile?  (I.e., why are we depending on the test's own post script to restore the environment?...shouldn't the testing superstructure or the next test's pre script be doing that?)

Reply:
It is.  It is also in `b0.69` and we are not going to merge this code.